jatszohaz/jatszohaz/models.py

```diff
@@ -110,3 +111,11 @@ def reset_all_checked_profile():
             user.room = ""
             user.checked_profile = False
             user.save()
+
+
+class UserComment(TimeStampedModel):
+    """Represents a comment for a user object made by a member (who is also a user but with permissions)."""
+    user = models.ForeignKey(JhUser, on_delete=models.PROTECT, related_name='user')
+    creator = models.ForeignKey(JhUser, on_delete=models.PROTECT, related_name="creator")

```

Comment:
Ez múltkor nem tűnt fel, de itt a a `related_name` mező használata szerintem nem a legjobb. Ez azt adja meg, hogy a JhUser felől milyen néven lehet elérni a UserComment objektumokat (tehát a reláció másik odala).

Pl. valami ilyesmi, ami az első felhasználó által létrehozott UserComment objektumokat adja vissza:
`JhUser.objects.first().creator.all()`
Egy lehetséges átnevezés, de lehet van jobb név is rá:
```suggestion
    user = models.ForeignKey(JhUser, on_delete=models.PROTECT, related_name='comments')
    creator = models.ForeignKey(JhUser, on_delete=models.PROTECT, related_name='created_user_comments')
```